### DISCUSSION TITLE
<title> code color

### DISCUSSION THREAD

**Discussion by arychi** [community/community#61823] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

it is hard for me to discriminate between code and hashtags in the code box. Is there a way that I can change these colors?

**Comment by BaraaBasim** [comment]
Yes, you can go to Settings -> Appearance -> Markdown editor font preference 
Font preference for plain text editors that support Markdown styling (e.g. pull request and issue descriptions, comments.)

**Comment by arychi** [comment]
it doesn't have any option to choose between! what can I do then?



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality11&>
08/01/23,
03:25:11 PM
